### ISSUE TITLE
Unable to Render File

### ISSUE THREAD

**Issue by pgrover-Row64** [Row64/Row64_Issue_Tracker#5]
****Row64 Version:2.0**

**Operating System & Hardware:** 11th Gen Intel(R) Core(TM) i7-11800H @ 2.30GHz   2.30 GHz 64-bit operating system, x64-based processor

**Description:**
Since last Thursday I have been unable to render files (attached). I was able to render a file previously. 
I have tried trouble shooting on my end. I have uninstalled and reinstalled version 1.5 & 2.0. I have updated the NVIDA download drivers. There was a formula in my data frame, which I also removed. Please also see screen grab of the issue. 

![Render Issue](https://user-images.githubusercontent.com/116025283/196251529-a7802ca3-1e4a-45ba-bdc6-5394695cfd6f.png)

[Museum (2).zip](https://github.com/Row64/Row64_Issue_Tracker/files/9803402/Museum.2.zip)

Thank you!

**Comment by onlysufyankhan** [comment]
Hey, Did you resolve the issue? i am having the same issue here.

**Comment by misner5** [comment]
Ok cheers...  We've done some testing on this with version 3.2, which can be downloaded here:
https://app.row64.com/Downloads/

That version has a massive number of improvements over 2.0.  
However I can still reproduce this problem.  We've raised the priority on this bug and will get back with more details...

**Comment by onlysufyankhan** [comment]
Thanks Man. Unfortunately version 3.2 have same rendering issue. Blank screen only.

**Comment by misner5** [comment]
Ok well... I can confirm we figured out what the bug was and fixed it internally.

We added a bunch of new low graphics capabilities that make Row64 faster with very large data sets.
That update had a side effect of causing problems in some animated charts... we do test animated charts in our release testing process but all our tests worked and this bug slipped by.

Anyways this fix will be part of Version 3.3... it's a ways off we are still finishing up this release but I will post here when it's available.

Cheers and thanks for your help and diligence finding this issue
